### ISSUE TITLE
windows build appears broken

### ISSUE THREAD

**Issue by rconde01** [electron/build-tools#264]
Hard to say if it's the same underlying issue but https://github.com/electron/build-tools/issues/261 is still broken for me.

I'm using the following docker config (note I don't think this has anything to do with docker...it's just what i'm using):
```
FROM mcr.microsoft.com/windows:2004

ENV chocolateyUseWindowsCompression false

RUN powershell -Command Set-ExecutionPolicy Unrestricted -Force
RUN powershell -Command \
    iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1')); \
    choco feature disable --name showDownloadProgress

RUN choco install -y -d --accept-license git
RUN git config --system core.longpaths true

RUN choco install -y -d --accept-license nodejs
RUN choco install -y -d --accept-license vim

# chrome build needs python2...eww
RUN choco install -y -d --accept-license python2
RUN pip install pywin32
RUN pip install crcmod

# https://chromium.googlesource.com/chromium/src/+/master/docs/windows_build_instructions.md#visual-studio
RUN choco install -y -d --accept-license visualstudio2019community \
    --package-parameters \
    "--add Microsoft.VisualStudio.Workload.NativeDesktop --add Microsoft.VisualStudio.Component.VC.ATLMFC --includeRecommended --passive"

# https://github.com/electron/build-tools
RUN npm install --global yarn
RUN npm i -g @electron/build-tools

# Setup required git settings for electron build tools
# If you don't set these the e init will fail
RUN git config --global core.filemode false
RUN git config --global core.autocrlf false
RUN git config --global branch.autosetuprebase always

RUN e init --root=C:/electron release
RUN e sync -vvvv

CMD [ "cmd" ]
```

FYI I need to increase the default vm size because 20GB isn't enough. I did this through editing `"C:\ProgramData\Docker\config\daemon.json"` to:
```
{
  "registry-mirrors": [],
  "insecure-registries": [],
  "debug": false,
  "experimental": false,
  "storage-opts": [
    "size=60GB"
  ]
}
```
and restarting docker.

Then you have to build with a larger memory because the default isn't enough.
`docker build --memory 8g electron_builid_win .`

After that, run the container:
`docker run --memory 8g -it electron_builder_win`

And try to build:
`e build electron:dist`

and get:
```
Writing new goma.gn file C:\Users\ContainerAdministrator\.electron_build_tools\third_party\goma.gn
Downloading "https://electron-build-tools.s3-us-west-2.amazonaws.com/build-dependencies/064e0beb3c11b82b66ae4b4959e6c0cdaa2116d77c50841e4a213d4f4fb1aeaf/goma-win.zip" into C:\Users\ContainerAdministrator\.electron_build_tools\third_party\goma-win.zip
[=======================================================================================================================================================================================================] 9266800/bps 100% 0.0s 
Running "goma_ctl.py ensure_start" in C:\Users\ContainerAdministrator\.electron_build_tools\third_party\goma
ERROR Error: Command failed: C:\Windows\system32\cmd.exe /d /s /c "vpython goma_ctl.py ensure_start"
'vpython' is not recognized as an internal or external command,
operable program or batch file.

    at checkExecSyncError (node:child_process:690:11)
    at Object.execFileSync (node:child_process:709:15)
    at Object.ensureGomaStart [as ensure] (C:\Users\ContainerAdministrator\.electron_build_tools\src\utils\goma.js:175:16)
    at runNinja (C:\Users\ContainerAdministrator\.electron_build_tools\src\e-build.js:60:10)
    at Object.<anonymous> (C:\Users\ContainerAdministrator\.electron_build_tools\src\e-build.js:133:3)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
```

**Comment by rzhao271** [comment]
I have https://github.com/electron/build-tools/pull/265 out for review right now. It should revert things to how they were around two weeks ago for Windows users.

Meanwhile, a workaround is going into the goma directory (`C:\Users\user\.electron_build_tools\third_party\goma`) on cmd and running 
`goma_ctl.bat ensure_start`

**Comment by rconde01** [comment]
ok - i went to try that workaround and when i ran e build electron:dist I get:
```
Checking for build-tools updates
Running "git pull --rebase --autostash" in C:\Users\ContainerAdministrator\.electron_build_tools
From https://github.com/electron/build-tools
   3a56b0d..4895bfd  master     -> origin/master
Applying autostash resulted in conflicts.
Your changes are safe in the stash.
You can run "git stash pop" or "git stash drop" at any time.
Successfully rebased and updated refs/heads/master.
Running "npx yarn" in C:\Users\ContainerAdministrator\.electron_build_tools
Updated to Latest Build Tools
C:\Users\ContainerAdministrator\.electron_build_tools\src\e:1
<<<<<<< Updated upstream
^^

SyntaxError: Unexpected token '<<'
    at Object.compileFunction (node:vm:355:18)
    at wrapSafe (node:internal/modules/cjs/loader:1022:15)
    at Module._compile (node:internal/modules/cjs/loader:1056:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47
```

Not sure if I should open a separate issue for that?

**Comment by rzhao271** [comment]
I think that just means you had a merge conflict that wasn't fixed.
I do not see the string `<<<<` show up on my PR.

**Comment by rconde01** [comment]
hmmm...I didn't change anything 🤷 I'm just pulling everything again

**Comment by rzhao271** [comment]
Are you using goma by any chance?
When I went to `src` (the directory containing the Chromium source) and then typed `gn args out/Testing`, I noticed that the file included a directory, but used single backslashes instead of double backslashes.
Manually changing the backslashes, and then manually issuing `ninja -C out\Testing -j 200 electron` from that same `src` directory did the trick. Using `e build` instead at this point would result in `e` trying to revert the args file back to using single backslashes.
